src/components/entity-info.component.js

```diff
@@ -124,7 +124,7 @@ export const EntityInfo = ({ entity, orgs, language, navigation }: Props) => {
           subtitle={entity.email}
           subtitleStyle={styles.listSubTitle}
           onPress={() =>
-            Communications.email([entity.email], null, null, 'Hi!', '')}
+            Communications.email([entity.email], null, null, null, '')}
```

Comment:
There you go: https://github.com/gitpoint/git-point/pull/229/commits/14d719fee6894eb98d3a2ec45e40cd046e1a477a
Thanks for the quick review :)